rheem-core/src/main/java/org/qcri/rheem/core/function/FunctionDescriptor.java

```diff
@@ -7,10 +7,7 @@
 
 import java.io.Serializable;
 import java.util.Optional;
-import java.util.function.BiFunction;
-import java.util.function.BinaryOperator;
-import java.util.function.Function;
-import java.util.function.Predicate;
+import java.util.function.*;
```

Comment:
It's generally bad practice to use the wild card import *. It can cause confusions and errors in the future. Also, it's not sublime/vim friendly :)